src/main/java/nextstep/courses/domain/session/Session.java

```diff
@@ -4,40 +4,39 @@
 import nextstep.payments.domain.Payment;
 import nextstep.users.domain.NsUser;
 
+import java.time.LocalDateTime;
 import java.util.Set;
 
 public abstract class Session {
 
-    protected Long id;
+    protected long id;
     protected SessionBody sessionBody;
     protected SessionEnrollment sessionEnrollment;
 
-    protected Session(Long id, SessionBody sessionBody, SessionEnrollment sessionEnrollment) {
+    protected Session(long id, SessionBody sessionBody, SessionEnrollment sessionEnrollment) {
         this.id = id;
         this.sessionBody = sessionBody;
         this.sessionEnrollment = sessionEnrollment;
     }
 
     abstract public void enroll(NsUser nsUser, Payment payment);
 
+    abstract public long getFee();
```

Comment:
[컨벤션](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers) 확인 부탁드려요 
```suggestion
    public abstract long getFee();
```

Reply:
네 반영하겠습니다. 
자료 참고하겠습니다.  감사합니다.